### ISSUE TITLE
Failed to Process RHEL6 STIG from Tenable.io XCCDF

### ISSUE THREAD

**Issue by zummarius** [Vulnerator/Vulnerator#172]
#### Prerequisites
_Before submitting a new issue, please ensure you have completed the following (delete the space in the box and replace it with an "x" to denote that it has been completed)_

- [X] I have ensured that I am running the latest release
- [X] The issue is repeatable
- [X] The issue has not already been reported

#### Description
I ran the most recent version of the RHEL6 SCAP using tenable.io and downloaded the output XCCDF xml file.

When I tried to import and execute the xml, the ingest failed and the result is an xlsx file with no content.

#### To Reproduce
1. Go to tenable.io
2. Create a SCAP and OVAL Auditing scan using the following SCAP:
     SCAP File (zip):  U_RedHat_6_V1R23_STIG_SCAP_1-2_Benchmark.zip
     SCAP Data Stream ID:  scap_mil.disa.stig_datastream_U_RedHat_6_V1R23_STIG_SCAP_1-2_Benchmark
     SCAP Benchmark ID:  xccdf_mil.disa.stig_benchmark_RHEL_6_STIG
     SCAP Profile ID: xccdf_mil.disa.stig_profile_MAC-2_Sensitive
     OVAL Result Type:  Full results w/ system characteristics
3. Run the scan on the target device.
4. Download the linux-0-sccdf-res.xml file upon completion of the scan.
5.  Import the linux-0-sccdf-res.xml file into Vulnerator.
6.  Click on "Execute"
7.  Observe that the STATUS column shows "Failed; See Log".

#### Expected Outcome
XCCDF File should have been properly processed and a completed xlsx file created.

#### Actual Outcome
Execution of the XCCDF file failed

#### System Information
 - _**OS: Windows 10
 - _**Version:  v6.1.9

#### Additional Context
N/A

#### Attachments

- [ ] Vulnerator Log (required for bugs) - this can be found at

**Comment by Zeluha** [comment]
I posted this on the .nessus export issue as well, but I believe this is because of the Tenable.io XCCDF export not having the correct context to parse compared to the SCC version. 

It looks like SCC always includes the referenced descriptions of every checked item and fix text included for each reference. The Tenable.io export only shows the raw value of pass or fail with no description outside of the title (all 1 block of text).  

Examples below of the differences (I am aware one of them is months old, but comparing for the formatting differences):

**Tenable.io export:** 
`<xccdf:rule-result role="full" version="WN10-UR-000045" idref="xccdf_mil.disa.stig_rule_SV-78349r1_rule" severity="high" weight="10.0">

<xccdf:result>pass</xccdf:result>

<xccdf:ident system="http://iase.disa.mil/cci">CCI-002235</xccdf:ident>


<xccdf:check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" negate="false">

<xccdf:check-content-ref href="#oval1" name="oval:mil.disa.fso.windows:def:4249"/>

</xccdf:check>

</xccdf:rule-result>`


**SCC export:**
`<cdf:rule-result idref="xccdf_mil.disa.stig_rule_SV-78349r1_rule" severity="high" weight="10.0" version="WN10-UR-000045" time="2019-07-15T12:00:25">

<cdf:result>pass</cdf:result>

<cdf:ident system="http://iase.disa.mil/cci">CCI-002235</cdf:ident>

<cdf:fix id="F-69787r2_fix"/>


<cdf:check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">

<cdf:check-content-ref href="#scap_mil.disa.stig_comp_U_MS_Windows_10_V1R14_STIG_SCAP_1-2_Benchmark-oval.xml" name="oval:mil.disa.fso.windows:def:4249"/>

</cdf:check>

</cdf:rule-result>`


I have a feeling this is going to cause a huge issue of parsing differences since the references are being lost in the Tenable.io XCCDF to give a nice report-style export. I could be incorrect though... I was hoping the .nessus export would work, but it looks like ACAS doesn't like to export compliance checks with full information in any working style other than PDF (The CSV exports are blank entirely as compliance doesn't have a "vulnerability critical level" as vulnerabilities do in tenable).

**Comment by amkuchta** [comment]
@Zeluha you are correct on all fronts with regards to the data provided by Tenable.io - the reports are not as complete as those generated by SCC and they are formatted slightly differently.

That being said, I have built a new method of ingesting STIGs into Vulnerator into the next version, which can be used to get around this issue.  Once I get the next release out, I will look at adding a processor for handling XCCDF files generated by Tenable.

Thank you for submitting the request, @zummarius!
